Sources/TidepoolKit/Internal/Extensions/UserDefaults.swift

```diff
@@ -29,6 +30,15 @@ extension UserDefaults {
             set(try? JSONEncoder.tidepool.encode(newValue), forKey: Key.defaultEnvironment.rawValue)
         }
     }
+    
+    public var studyProductSelection: String? {
+        get {
+            string(forKey: Key.studyProductSelection.rawValue)
+        }
+        set {
+            set(newValue, forKey: Key.studyProductSelection.rawValue)
+        }
+    }
```

Comment:
I believe this repo is just the interface to Tidepool Backend. I think the repo name is confusing, since it is not a general kit for Tidepool. Probably should be named `TidepoolBackendKit` or similar. 

Since this is a common repo, I think this code is better placed elsewhere. I'm starting to understand better why you originally wanted it in LoopKit, since you need it in a common repo. Wonder if we should chat about this?